openedx/features/survey_report/application.py

```diff
@@ -0,0 +1,6 @@
+"""
+Contains the logic for generating a survey report.
```

Comment:
maybe we can replace "for generating" to "for manage" because we will write the send method here too.